event-handlers/src/test/java/no/sikt/nva/nvi/events/cristin/CristinMapperTest.java

```diff
@@ -1,16 +1,98 @@
 package no.sikt.nva.nvi.events.cristin;
 
+import static no.sikt.nva.nvi.events.cristin.CristinMapper.AFFILIATION_DELIMITER;
+import static no.sikt.nva.nvi.events.cristin.CristinMapper.API_HOST;
+import static no.unit.nva.testutils.RandomDataGenerator.randomString;
+import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertThrows;
+import java.math.BigDecimal;
+import java.net.URI;
 import java.util.List;
+import no.sikt.nva.nvi.common.service.model.PublicationDetails.PublicationDate;
+import nva.commons.core.paths.UriWrapper;
 import org.junit.jupiter.api.Test;
 
 class CristinMapperTest {
 
+    public static final double POINTS_PER_CONTRIBUTOR = 2.0;
+
     @Test
     void shouldThrowNullPointerExceptionWhenQualityCodeIsMissing() {
         var empty = emptyScientificResource();
-        var build = CristinNviReport.builder().withScientificResources(List.of(empty)).build();
-        assertThrows(NullPointerException.class, () -> CristinMapper.toDbCandidate(build));
+        var report = CristinNviReport.builder().withScientificResources(List.of(empty)).build();
+        assertThrows(NullPointerException.class, () -> CristinMapper.toDbCandidate(report));
+    }
+
+    @Test
+    void shouldUseCristinLocaleInstitutionWhenSummarizingPoints() {
+        var institutionIdentifier = randomString();
+        var creators = List.of(scientificPersonAtInstitutionWithPoints(institutionIdentifier, POINTS_PER_CONTRIBUTOR),
+                               scientificPersonAtInstitutionWithPoints(institutionIdentifier, POINTS_PER_CONTRIBUTOR));
+        var scientificResource = scientificResourceWithCreators(creators);
+        var cristinLocale = cristinLocaleWithInstitutionIdentifier(institutionIdentifier);
+        var report = cristinReportFromCristinLocalesAndScientificResource(cristinLocale, scientificResource);
+        var dbCandidate = CristinMapper.toDbCandidate(report);
+
+        var institutionId = dbCandidate.points().get(0).institutionId();
+        var expectedInstitutionId = constructExpectedInstitutionId(cristinLocale);
+        var expectedPointsForInstitution = BigDecimal.ZERO
+                                               .add(new BigDecimal(POINTS_PER_CONTRIBUTOR))
+                                               .add(new BigDecimal(POINTS_PER_CONTRIBUTOR)).setScale(1);
```

Comment:
In rest of nvi, the scale is 10 during calculations and 4 when persisting result of calculations. Maybe do the same here?

Reply:
These values contain many decimals